test/e2e/reconciler/cluster/controller_test.go

```diff
@@ -53,7 +53,11 @@ const sourceClusterName, sinkClusterName = "source", "sink"
 
 func TestClusterController(t *testing.T) {
 	t.Parallel()
-	framework.Suite(t, "transparent-multi-cluster")
```

Comment:
Can you explain a bit more the reason of this change ?

Reply:
The test fails without kind, I forgot exactly why. I'm reverting this change for now.

Reply:
I now remember why. I wonder what's the purpose of running the tests against a fake pcluster. Why not running tests always against a real pcluster? It seems to be it's a lot of code to maintain for little value ?